python/openai/README.md

```diff
@@ -315,3 +315,15 @@ available arguments and default values.
 
 For more information on the `tritonfrontend` python bindings, see the docs
 [here](https://github.com/triton-inference-server/server/blob/main/docs/customization_guide/tritonfrontend.md).
+
+## Known Limitations
+
+- For multi-GPU models (ex: Tensor Parallelism > 1), here is the current support matrix:
+  - [x] vLLM ([EngineArgs](https://github.com/triton-inference-server/vllm_backend/blob/main/README.md#using-the-vllm-backend))
+    - ex: Configure `tensor_parallel_size: 2` in the
+      [model.json](https://github.com/triton-inference-server/vllm_backend/blob/main/samples/model_repository/vllm_model/1/model.json)
+  - [x] TensorRT-LLM ([Orchestrator Mode](https://github.com/triton-inference-server/tensorrtllm_backend/blob/main/README.md#orchestrator-mode))
+    - `export TRTLLM_ORCHESTRATOR=1`
+    - **NOTE**: Orchestrator mode does not currently support multi-node models
```

Comment:
I think we would need to define this - I think people will get confused with multi-node and multi-gpu ....

Do we need to mention multi-gpu ?

Reply:
I wanted to be clear about the limitations of orchestrator mode, but it's mentioned in the linked orchestrator docs, so I think we could remove it if you're concerned about mentioning it:
> Since this mode uses [MPI_Comm_spawn](https://www.open-mpi.org/doc/v4.1/man3/MPI_Comm_spawn.3.php), it might not work properly with [slurm](https://slurm.schedmd.com/) deployments. Additionally, this currently only works for single node deployments.

Reply:
As currently written - I'd leave it out of here - I think the relationship between multi-node and multi-gpu would need more clarification to be useful - I would lean on the orchestrator mode docs.

Reply:
Removed in favor or orchestrator docs